WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -701,20 +791,80 @@ public void onFailure(int statusCode) {
     }
 
     /*
+     * post slugs resolution to IDs has completed
+     */
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ReaderEvents.PostSlugsRequestCompleted event) {
```

Comment:
@nbradbury just to be clear: in order to do that check we need to keep the slugs in the `ReaderPostDetailFragment` since the post is not actually displayed yet and we're waiting for the slug resolution to finish. I specifically removed any slug references from the fragment to cut back on the "slug pollution" so, are you sure you want to implement the check. Besides, there is only one place the event is emitted and that when the parent activity's slug-resolution-request finishes and no post is displayed in the fragment anyway. Is there some other cases to cover? Thanks!